TanukiKit/Repositories.swift

```diff
@@ -5,60 +5,459 @@ import RequestKit
+    open var accessLevel: Int?
+    open var notificationLevel: Int?
+
+    public init(_ json: [String: AnyObject]) {
+        accessLevel = json["access_level"] as? Int
+        notificationLevel = json["notification_level"] as? Int
+    }
+}
+
+//@objc open class EventData: NSObject {
+//    open var before: String?
+//    open var after: String?
+//    open var ref: String?
+//    open var userID: Int?
+//    open var userName: String?
+//    open var repository: Repository? // This is one of the cases where it causes conflict.
+//    open var project: Repository? // TODO: Change to project
+//    open var commits: [Commit]? // TODO: Implement Commit GET request.
+//    open var totalCommitsCount: Int?
+//
+//    public init(_ json: [String: AnyObject]) {
+//        before = json["before"] as? String
+//        after = json["after"] as? String
+//        ref = json["ref"] as? String
+//        userID = json["user_id"] as? Int
+//        userName = json["user_name"] as? String
+//        repository = Repository1(json["repository"] as? [String: AnyObject] ?? [:]) // TODO: Change to repository
+//        project = Repository(json["project"] as? [String: AnyObject] ?? [:]) // TODO: Change to project
+//        totalCommitsCount = json["total_commits_count"] as? Int
+//    }
+//}
+
+//@objc open class Note: NSObject {
+//    open var id: Int?
+//    open var body: String?
+//    open var attachment: String?
+//    open var author: User?
+//    open var createdAt: Date?
+//    open var system: Bool?
+//    open var upvote: Bool?
+//    open var downvote: Bool?
+//    open var noteableID: Int?
+//    open var noteableType: String?
+//
+//    public init(_ json: [String: AnyObject]) {
+//        id = json["id"] as? Int
+//        body = json["body"] as? String
+//        attachment = json["attachment"] as? String
+//        author = User(json["author"] as? [String: AnyObject] ?? [:])
+//        createdAt = Time.rfc3339Date(string: json["created_at"] as? String)
+//        system = json["system"] as? Bool
+//        upvote = json["upvote"] as? Bool
+//        downvote = json["downvote"] as? Bool
+//        noteableID = json["noteable_id"] as? Int
+//        noteableType = json["noteable_type"] as? String
+//    }
+//}
+
 public extension TanukiKit {
     /**
-     Fetches the Repositories for the current user
-     - parameter owner: The user or organization that owns the repositories. If `nil`, fetches repositories for the authenticated user.
-     - parameter page: Current page for repository pagination. `1` by default.
-     - parameter perPage: Number of repositories per page. `100` by default.
+     Fetches the Projects for which the authenticated user is a member.
+     - parameter page: Current page for project pagination. `1` by default.
+     - parameter perPage: Number of projects per page. `100` by default.
+     - parameter archived: Limit by archived status. Default is `""`, set to `true` to enable.
```

Comment:
If I understand this right I have to write:

```swift
client.repositories(..., archived: "true", ...)
```

This seems very weird to me from an API usage perspective. It would be helpful to use the GitLab default. This means if I make a request to the API and don't supply the `archived` parameter do I get archived projects or not? Whatever the default is we should default to that too.